src/codecs/ipynb/__file_snapshots__/meta-analysis.json

```diff
@@ -15119,7 +15109,7 @@
         "execution_count": 12
       }
     },
-    null,
+    0,
```

Comment:
@nokome, Mind double checking this please? I'm not quite sure where this could be coming from and am having issues running the full test suite locally. Please merge this PR if it looks good, thanks.